understanding/language-of-parts.html

```diff
@@ -523,11 +437,8 @@ <h2>Key Terms</h2>
                </details>
             </section>
             <section id="test-rules">
-               <h2>Test Rules</h2>
-               <p>The following are Test Rules for certain aspects of this Success Criterion. It is
-                  not necessary to use these particular Test Rules to check for conformance with WCAG,
-                  but they are defined and approved test methods. For information on using Test Rules,
-                  see <a href="understanding-act-rules.html">Understanding Test Rules for WCAG Success Criteria</a>.
+               <h2>テストルール</h2>
+               <p>以下は、この達成基準の特定の側面に関するテストルールである。この特定のルールを使用して WCAG に適合しているかどうかを確認する必要はないが、これらのルールは定義され、承認されたテスト方法である。テストルールの使用については、<a href="understanding-act-rules.html">WCAG 達成基準のテストルールを理解する</a>を参照のこと。
                </p>
                <ul>
                   <li><a href="/WAI/standards-guidelines/act/rules/de46e4/">Element with lang attribute has valid language tag</a></li>
```

Comment:
テストルールのリンク先 URL が相対パスになっていたので、「https://www.w3.org/」を追記しました。

```suggestion
                  <li><a href="https://www.w3.org/WAI/standards-guidelines/act/rules/de46e4/">Element with lang attribute has valid language tag</a></li>
```

Reply:
もうひとつのリンクも

`<a href="https://www.w3.org/WAI/standards-guidelines/act/rules/off6ek/proposed/">HTML element language subtag matches language</a>` としていただけますと幸いです。

// なぜか修正提案ができませんでした...

Reply:
修正しました。